Data-Explorer/ui.R

```diff
@@ -6,15 +6,7 @@
 ### Original Date: October 2017
 
 
-#source("Data-Explorer/ui_code.R")
-
-### Visual interface ----
-#if (credentials$authorisation=="Private"){
-###Update each quarter
-publication_date<-"23-february-2021"
-
-#secure_app(
-  tagList( #needed for shinyjs
+ui_code <- tagList( #needed for shinyjs
   navbarPage(id = "Panels", # id used for jumping between tabs
              title = div(style = "position: relative; top: -5px;"),  
              #title for browser tab
```

Comment:
is the comment below about the css files needing to be removed for deployment still relevant/correct?